assets/layout/common/_footer.scss

```diff
@@ -136,7 +139,12 @@
   .youtube-link:active,
   .youtube-link:focus {
     background: $color-youtube;
-    padding: 18.5px 14.704px; // (55px - 18px) / 2 || (55px - (213.264px * (18 / 150))) / 2
+    padding: (($icon-size - 18px) / 2) (($icon-size - (213.264px * (18 / 150))) / 2);
```

Comment:
What is this doing? It's really hard to understand! I think a variable here (or just a px value) would be better than this complexity.